memegen/settings.py

```diff
@@ -28,6 +30,8 @@ class TestConfig(Config):
     DEBUG = True
     TESTING = True
 
+    SQLALCHEMY_DATABASE_URI = "postgresql://localhost/memegen_test"
```

Comment:
Thanks for the tip! For now, I'm just going to call `createdb memegen_test` before running tests. This is still a work in progress.